Longest_Common_Subsequence/main.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Author: Yash Ladha
+ * Topic : Longest Common Subsequence
+ * Domain: Dynamic Programming
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+void longestCommonSubsequence(char *string1, char *string2);
+
+int max(int x, int y);
+
+int main() {
```

Comment:
better move main() below the function. That way we won't have to use prototypes.